### ISSUE TITLE
Error with group option

### ISSUE THREAD

**Issue by nirav1911** [brianium/paratest#164]
I am getting an error when I use --group with paratest and some of the test cases don't have any group defined in the tests.

Error: Maybe there is more than one class in this file.

**Comment by sormy** [comment]
What version do you have?

**Comment by jaredmcateer** [comment]
I have this problem as well. I'm using the latest dev-master via composer as of today. I tried to exclude a group and I am getting this error on a file that has the annotation:

     paratest -p4 --configuration phpunit.xml --exclude-group=synchronousTests Website/

    Running phpunit in 4 processes with /data/istock/vendor/bin/phpunit

    Configuration read from /data/istock/ZF/unittest/phpunit.xml
                                                                                                                                                                                                                                              
    [RuntimeException]                                                                                                                                                                                                                          
    The process: '/data/www/vendor/bin/phpunit' '--configuration' '/data/www/ZF/unittest/phpunit.xml' '--exclude-group' 'synchronousTests' '--log-junit' '/tmp/PT_dCj3MU' 'Website_Asset_NamespaceTest' 'Website/Asset/NamespaceTest.php'                                                                                                                                                                                                                                           
    Log file "/tmp/PT_dCj3MU" is empty.                                                                                                                                                                                                         
    This means a PHPUnit process was unable to run "Website/Asset/NamespaceTest.php"                                                                                                                                                            
    Maybe there is more than one class in this file.

**Comment by sormy** [comment]
Could you write fail phpunit test for that case?

**Comment by jaredmcateer** [comment]
Sure thing

    paratest -p4 --exclude-group=skip MyTest.php

Works:
```php
<?php
class MyTest extends PHPUnit_Framework_TestCase
{
    /**
     * @group skip
     */
    public function testGroup()
    {
        $this->assertTrue(true);
    }

    public function testGroup2()
    {
        $this->assertTrue(true);
    }
}
```

Fails:
```php
<?php
class MyTest extends PHPUnit_Framework_TestCase
{
    /**
     * @group skip
     */
    public function testGroup()
    {
        $this->assertTrue(true);
    }

    /**
     * @group skip
     */    
    public function testGroup2()
    {
        $this->assertTrue(true);
    }
}
```

Fails:
```php
<?php
/**
* @group skip
*/
class MyTest extends PHPUnit_Framework_TestCase
{

    public function testGroup()
    {
        $this->assertTrue(true);
    }

    public function testGroup2()
    {
        $this->assertTrue(true);
    }
}
```

**Comment by jaredmcateer** [comment]
Further testing, if you try to test a group where none exist in the file(s) you're testing you will get this error.

E.g., if you run

    paratest --group=doesnotexist MyTest.php

on any of the cases above you will also get the runtime error.

**Comment by jaredmcateer** [comment]
I see that there is another discussion about this https://github.com/brianium/paratest/issues/146

**Comment by tsikytchax** [comment]
Hi,

I just installed paratest on a Symfony project to speed up my thousands phpunit tests. I had the same problem and I solved giving the right path to phpunit. Here is the command I execute : 

php -d memory_limit=1024M vendor/brianium/paratest/bin/paratest -p 2 --phpunit vendor/phpunit/phpunit/phpunit --bootstrap app/bootstrap.test.php -c app/phpunit.xml --testsuite AriaUnitTestSuite --log-junit aria-junit.xml --coverage-html ./ariaCodeCoverage --coverage-clover aria-clover.xml

May it help you!
Tcx
